osmoutils/accum/prefix.go

```diff
@@ -0,0 +1,30 @@
+package accum
+
+import "fmt"
+
+const (
+	modulePrefix      = "accum"
+	accumulatorPrefix = "acc"
+	positionPrefix    = "pos"
+)
+
+// formatAccumPrefix returns the key prefix used for any
+// accum module values to be stored in the KVStore.
+// Returns "accum/{key}" as bytes.
+func formatModulePrefixKey(key string) []byte {
+	return []byte(fmt.Sprintf("%s/%s", modulePrefix, key))
+}
+
+// formatAccumPrefix returns the key prefix used
+// specifically for accumulator values in the KVStore.
+// Returns "accum/acc/{name}" as bytes.
+func formatAccumPrefixKey(name string) []byte {
+	return formatModulePrefixKey(fmt.Sprintf("%s/%s", accumulatorPrefix, name))
+}
+
+// formatPositionPrefixKey returns the key prefix used
+// specifically for position values in the KVStore.
+// Returns "accum/pos/{accumName}/{address}" as bytes.
+func formatPositionPrefixKey(accumName, address string) []byte {
+	return formatAccumPrefixKey(fmt.Sprintf("%s/%s/%s", positionPrefix, accumName, address))
```

Comment:
@AlpinYukseloglu  I realized that the position prefix should also include the accumulator name so that positions are not applied to all accumulators.

This was caught in tests and is now fixed

Reply:
Great catch – I did find it odd that we only needed the address (and not the accumulator) to get a position, so this resolves my concerns there.